hello-world.php

```diff
@@ -10,4 +10,12 @@ function helloWorld($name)
     //
     // YOUR CODE GOES HERE
     //
+if($name !=""){
+echo "Hello,". $name."!";
 }
+else 
+echo "Hello World!";
+
+}
+helloWorld(mohamad);
```

Comment:
`PHP Notice:  Use of undefined constant mohamad - assumed 'mohamad' in /home/travis/build/simplonco/php-hello-world/hello-world.php on line 20`